_contentTemplates/common/rebind-method.md

```diff
@@ -0,0 +1,5 @@
+#intro
+You can refresh the data by invoking the `Rebind` method. Use the component reference to call the `Rebind` method after you have made the data changes. Thus, the component will process the available data anew to reflect the updates.
```

Comment:
Consider a link to the common Rebind documentation here. We can also mention here when is Rebind() necessary (same collection reference) and when not (new collection reference).